<launch>

<node pkg="tango_streamer" type="image_server.py" name="image_server_color">
   <param name="port_number" value="11111" type="int"/>
   <param name="camera_name" value="color_camera"/>
</node>

<node pkg="tango_streamer" type="image_server.py" name="image_server_fisheye">
   <param name="port_number" value="11115" type="int"/>
   <param name="camera_name" value="fisheye_camera"/>
</node>

<node pkg="tango_streamer" type="command_relay.py" name="command_relay" output="screen"/>

<node pkg="tango_streamer" type="intrinsics_server.py" name="intrinsics_server_color" output="screen">
   <param name="port_number" value="11114" type="int"/>
   <param name="camera_name" value="color_camera"/>
</node>

<node pkg="tango_streamer" type="intrinsics_server.py" name="intrinsics_server_fisheye">
   <param name="port_number" value="11116" type="int"/>
   <param name="camera_name" value="fisheye_camera"/>
</node>

<node pkg="tango_streamer" type="pointcloud_server.py" name="pointcloud_server"/>

<node pkg="tango_streamer" type="pose_server.py" name="pose_server" output="screen">
   <param name="port_number" value="11113" type="int"/>
   <param name="pose_topic" value="tango_pose"/>
   <param name="coordinate_frame" value="odom"/>
</node>

<node pkg="tango_streamer" type="phone_type_server.py" name="phone_type_server" output="screen">
</node>

<!-- <node pkg="tango_streamer" type="pose_area_server.py" name="pose_area_server">
   <param name="port_number" value="11117" type="int"/>
   <param name="pose_topic" value="tango_area_pose"/>
   <param name="coordinate_frame" value="area_learning"/>
</node> -->

<node pkg="image_transport" type="republish" name="decompress_fisheye" args="in:=fisheye_camera/image_raw compressed out:=fisheye_camera/image_raw raw"/>
<node pkg="image_transport" type="republish" name="decompress_color" args="in:=color_camera/image_raw compressed out:=color_camera/image_raw raw"/>
<node pkg="tango_streamer" type="rssi_server.py" name="RSSI"/>
<node pkg="tango_streamer" type="undistort_fisheye" name="undistort_fisheye">
  <param name="camera_name" value="fisheye_camera"/>
</node>
</launch>
